src/vscode-tree-item/vscode-tree-item.ts

```diff
@@ -142,6 +173,31 @@ export class VscodeTreeItem extends VscElement {
     this._mainSlotChange();
     this.role = 'treeitem';
     this.ariaDisabled = 'false';
+    this.addEventListener('pointerenter', this._handlePointerEnter);
+    this.addEventListener('pointerleave', this._handlePointerLeave);
+    this.addEventListener('focusin', this._handleFocusIn);
```

Comment:
Do we actually need these handlers? Maybe the `:focus-within` pseudo-class could handle this? A simple "no" is a perfectly fine answer.